crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -2660,44 +2660,40 @@ pub fn prepare_mesh_bind_group(
     // Create the skinned mesh bind group with the current and previous buffers
     // (the latter being for motion vector computation). If there's no previous
     // buffer, just use the current one as the shader will ignore it.
```

Comment:
Comment seems out of date.

```suggestion
    // (the latter being for motion vector computation).
```